profiles/rawls-sa/terraform/rawls/sa.tf

```diff
@@ -28,3 +42,18 @@ resource "vault_generic_secret" "app_account_key" {
   path = "${var.vault_path_prefix}/${var.service}/${var.service}-account.json"
   data_json = "${base64decode(google_service_account_key.app_account_key.private_key)}"
 }
+
+resource "google_service_account" "billingprobe" {
+  account_id   = "${var.owner}-billingprobe"
```

Comment:
note to self: in the terraformy version of the rawls conf, the config gets the SA email address from the SA key checked into vault by terraform